src/sisl/io/orca/stdout.py

```diff
@@ -258,9 +259,10 @@ def read_block(step_to):
 
         return read_block(step_to)
 
+    @units2dict
```

Comment:
This decorator needs to be the first decorator added. `SileBinder` should be the last since it changes the function to a class.